src/App.tsx

```diff
@@ -1,15 +1,56 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Fix it
<img width="913" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/dca6f4cd-6d6a-41ae-92d1-192aab278fc5">